include/zephyr/bluetooth/att.h

```diff
@@ -91,6 +91,12 @@ size_t bt_eatt_count(struct bt_conn *conn);
 
 #endif /* CONFIG_BT_EATT */
 
+enum bt_att_chan_pref {
+	BT_ATT_CHAN_ANY = 0x0,
+	BT_ATT_CHAN_UNENHANCED = BIT(0),
+	BT_ATT_CHAN_ENHANCED = BIT(1),
```

Comment:
Prefer to use the enum name as prefix for the fields:
```suggestion
	BT_ATT_CHAN_PREF_ANY = 0x0,
	BT_ATT_CHAN_PREF_UNENHANCED = BIT(0),
	BT_ATT_CHAN_PREF_ENHANCED = BIT(1),
```

Also consider to rename `ANY` to `NONE` similar to the other options field we use in BT, e.g. `BT_LE_ADV_OPT_NONE`.